tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.
+
+All activities of this TAG should aim to positively impact the community by changing mindset, increasing resource efficiency, and putting energy efficiency and environmental sustainability on everyone's agenda.
+
+## Responsibilities & Deliverables
+
+### Background
+We recognize that: 
+- balancing resource consumption with the desire for performance is difficult
+- there is inconsistency in how emissions are calculated and what is included by software in scopes of emission. 
+
+### In-Scope
+
+- Identify, define, and develop tooling to assess and improve environmental sustainability approaches, including
+    - quantification of energy consumption of full-stack software including application, cloud native services
+    - recommendations to manage full-stack software such that the overall energy consumption and associated carbon are reduced.
```

Comment:
"Recommendations and strategies to develop, package, distribute, deploy, and operate cloud native projects that reduce energy consumption and carbon as well as the various environments they operate in and which impact their consumption"

I recommend avoiding full stack here as it can limit your scope to purely operational deployments and not apply to the development.  Enumerating the various phases of software development and software operation here allows you to consider their impact on the next stage. Potentially identifying optimizations at each step that build on each other. Further, consumption on edge compute versus a laptop versus an EC2 instance all vary so environment of deployment and throughout make a huge difference."